src/_templates.scss

```diff
@@ -0,0 +1,16 @@
+@import 'variables.scss';
+
+%base_style {
+    background-color: $background_color;
+    color: $main_text_color;
+    font-family: Merriweather;
```

Comment:
I can't be confident that all operating systems can see this font by default. That's highly recommended to import this font locally or import it from some CDN.